tests/test_ectrans4py/test_ectrans4py.py

```diff
@@ -13,7 +13,7 @@
     raise NotImplementedError("ectrans4py does not support Windows")
 
 KNUMMAXRESOL = 10
-EPSILON = 1e-13
+EPSILON = 1e-10
```

Comment:
This is needed once we actually compare output against input rather than input against itself?

Reply:
Exactly. 1e-13 is too strict, at least when testing on ECMWF/AC with GCC, MKL etc. I don't know if this indicates faulty behaviour or if it's to be expected.